src-java/flowhs-topology/flowhs-messaging/src/main/java/org/openkilda/messaging/command/flow/FlowRerouteFlushRequest.java

```diff
@@ -35,12 +35,16 @@ public class FlowRerouteFlushRequest extends CommandData {
     @JsonProperty("flow_id")
     private String flowId;
 
+    private String flowType;
```

Comment:
No need to mark as private all the fields here since @Value annotation
@Value already marks non-static, package-local fields private.

Reply:
it is true, but I would prefer to use common style. Since we have it in other places with @value I keep it here as well

Reply:
fixed